images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -307,3 +307,9 @@ function Get-SwigVersion {
     $swigVersion = $Matches.Version
     return "Swig $swigVersion"
 }
+
+function Get-ImageMagickVersion {
+    ($(magick -version) | Select-String -Pattern "Version") -match "(?<version>\d+\.\d+\.\d+.\d+)" | Out-Null
```

Comment:
Maybe a bit easier to read but up to you
`"(?<version>(\d+\.){2}\d+-\d+)"`

Reply:
Partially accepted. The dot was replaced with the expected character.